x-pack/plugins/lists/server/plugin.ts

```diff
@@ -51,9 +57,15 @@ export class ListPlugin implements Plugin<ListPluginSetup, ListsPluginStart, {},
     initRoutes(router, config);
 
     return {
-      getExceptionListClient: (savedObjectsClient, user): ExceptionListClient => {
+      getExceptionListClient: (
+        savedObjectsClient,
+        user,
+        disableServerExtensionPoints = false
```

Comment:
one can turn off execution of extension points

Reply:
Do we want this on by default? (No strong opinion here, just curious what your thoughts are).

Reply:
IMO, yes. Server side extension are there to ensure that domain specific checks are done and having it turned off by default would mean that any consumer of the Lists plugin would need to be conscious of what they are doing when attempting to get an ExceptionListClient instance. I was actually a bit reluctant to add this capability, but did so based on feedback received during discussions around this feature.

Reply:
Don't most people and API's utilize booleans that are positive to avoid double negatives. For example, shouldn't this variable be:

```ts
enableServerExtensionPoints = true
```

I'm not strong about which way you want the boolean, but for the most part I try to avoid having double negatives and use true when something is enabled and false when something is disabled rather than flipping it the other way around.

Reply:
Thanks. I will make the change. Normally, I set boolean type of optional function arguments to false, thus why I originally went with `disable**`.  What you say makes sense 👍